api/app/signals/apps/api/generics/permissions.py

```diff
@@ -141,3 +141,11 @@ def has_object_permission(self, request, view, obj):
         )
 
         return bool(has_category_read_permission) and request.user.has_perm('signals.sia_read')
+
+
+class SIAReportPermissions(SIABasePermission):
+    perms_map = {
+        'GET': ['signals.sia_signal_report'],
```

Comment:
I'd add the general `signals.sia_read` permission as well (to match the other permissions).

Reply:
check